test/AspNet.Security.OAuth.Providers.Tests/LinkedIn/bundle.json

```diff
@@ -12,20 +12,48 @@
       }
     },
     {
-      "comment": "https://developer.linkedin.com/docs/signin-with-linkedin",
-      "uri": "https://api.linkedin.com/v1/people/~:(id,first-name,last-name,formatted-name,email-address)",
+      "comment": "https://docs.microsoft.com/en-us/linkedin/consumer/integrations/self-serve/sign-in-with-linkedin",
+      "uri": "https://api.linkedin.com/v2/me?projection=(id,firstName,lastName)",
       "contentFormat": "json",
       "contentJson": {
-        "firstName": "Frodo",
-        "headline": "Jewelery Repossession in Middle Earth",
         "id": "1R2RtA",
-        "lastName": "Baggins",
-        "formattedName": "Frodo Baggins",
-        "emailAddress": "frodo@shire.middleearth",
-        "siteStandardProfileRequest": {
-          "url": "https://www.linkedin.com/profile/view?id=frodo-baggins"
+        "firstName": {
+          "localized": {
+            "en_US": "Frodo"
+          },
+          "preferredLocale": {
+            "country": "US",
+            "language": "en"
+          }
+        },
+        "lastName": {
+          "localized": {
+            "en_US": "Baggins"
+          },
+          "preferredLocale": {
+            "country": "US",
+            "language": "en"
+          }
+        },
+        "profilePicture": {
+          "displayImage": "urn:li:digitalmediaAsset:B54328XZFfe2134zTyq"
```

Comment:
If this is being mapped, can you add it as a claims test case?

Reply:
I added a unit test to check this claim, using the sample JSON provided in LinkedIn documentation